extensions/amp-story/0.1/amp-story.js

```diff
@@ -558,16 +544,14 @@ export class AmpStory extends AMP.BaseElement {
     const previousActivePriorSibling = scopedQuerySelector(
         this.element, `[${PRE_ACTIVE_PAGE_ATTRIBUTE_NAME}]`);
 
-    this.maybeApplyFirstAnimationFrame_(targetPage);
-
     return this.mutateElement(() => {
       this.activePage_ = targetPage;
       this.triggerActiveEventForPage_();
       this.systemLayer_.resetDeveloperLogs();
       this.systemLayer_.setDeveloperLogContextString(
           this.activePage_.element.id);
-      this.maybeStartAnimations_(targetPage);
     })
+        .then(() => targetPage.beforeVisible())
```

Comment:
Does this need to be called from AmpStory? Why isn't calling this from the `layoutCallback` sufficient?

Reply:
Needs to be reapplied every time you navigate back to a page